Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1188,6 +1190,20 @@ acceptStaticOfferDriverRequest mbSearchTry driver quoteId reqOfferedValue mercha
       Just searchTry -> deactivateExistingQuotes booking.merchantOperatingCityId merchant.id driver.id searchTry.id $ mkPrice (Just quote.currency) quote.estimatedFare
       Nothing -> pure []
   void $ sendRideAssignedUpdateToBAP booking ride driver vehicle
+  now <- getCurrentTime
+  when booking.isScheduled $ do
+    let scheduledPickup = booking.fromLocation
+        scheduledTime = booking.startTime
+        pickupPos = LatLong {lat = scheduledPickup.lat, lon = scheduledPickup.lon}
+    void $ QDriverInformation.updateLatestScheduledBookingAndPickup (Just scheduledTime) (Just pickupPos) driver.id
+    maxShards <- asks (.maxShards)
+    let jobScheduledTime = (diffUTCTime booking.startTime now) - (30 * 60)
```

Comment:
lets make like like this `max 2 ((diffUTCTime booking.startTime now) - (30 * 60))`

Reply:
okay you mean if jobSCheduledTIme can't be less than 2 seconds from current time right?